### ISSUE TITLE
Failed to convert feature matrix X and label y to xlearn data format

### ISSUE THREAD

**Issue by dudududa007** [PKU-Cloud-Lab/xLearn#370]
I got an error when I run "example_FM_wine.py" file.

____________________

Failed to convert feature matrix X and label y to xlearn data format
During handling of the above exception, another exception occurred:
  File "D:\Src\example_FM_wine.py", line 50, in <module>
    fm_model.fit(X_train,
____________________

The same error occurs in"example_LR_iris.py" file.
The error does not occur in "example_FFM_criteo.py" file. (Because this file uses a txt file as a parameter when calling the fit function.)

how do I fix it?
Thanks a lot.
